pcsx2/USB/usb-mic/audiodev-pulse.cpp

```diff
@@ -334,7 +334,7 @@ namespace usb_mic
 			if (mPAready == 3 && dur >= 1000)
 			{
 				mLastGetBuffer = now;
-				int ret = pa_context_connect(mPContext,
+				[[maybe_unused]]int ret = pa_context_connect(mPContext,
```

Comment:
This is a bit of a mouthful to read, why not put maybe_unused on the line above? Do we have a guideline on same-line [[]] blocks?

Reply:
Don't think we have any sort of guideline on attributes yet, since we only recently started adding them to pcsx2.

Reply:
I went ahead and moved it up a line on this one file.